hs_core/management/commands/mongo_update.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from hs_core.hydro_realtime_signal_processor import update_mongo
+from hs_core.models import BaseResource
+
+
+class Command(BaseCommand):
+    help = "Update Mongo Discovery records."
+
+    def handle(self, *args, **options):
+        dqs = BaseResource.objects.filter(Q(raccess__discoverable=True)
+                                          | Q(raccess__public=True)).select_related('raccess')
+        print("Django count = {}".format(dqs.count()))
+        for r in dqs.iterator():
+            print("Updating Mongo record for " + r.short_id)
+            try:
+                update_mongo(r.short_id)
+            except Exception as e:
+                print("failed to update Mongo record for " + r.short_id)
+                print(e)
```

Comment:
might want to store the resources that fail to update and print them at the end of the command, otherwise might be hard to find them in the long list of resources?